app/resonant-laboratory/views/widgets/DatasetView/template.html

```diff
@@ -1,5 +1,5 @@
-<img id="noDatasetState" class="emptyStateHelper" src="../../../images/emptyState/noDataset.svg"/>
-<div id="datasetOverview">
+<img id="noDatasetState"  style="display: none;" class="emptyStateHelper" src="../../../images/emptyState/noDataset.svg"/>
```

Comment:
I tend to dislike doing this in the template if there's already logic in the javascript to show/hide pieces. But that's a personal style preference, so this is fine for now.